Updated tests for SfePy recipe.

BubuLK opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:

Added 'python run_test.py' (including necessary source files) to meta.yaml test section.
-->

BubuLK committed:
MNT: Re-rendered with conda-smithy 3.2.2 and pinning 2018.12.28

BubuLK committed:
Add samle "run_test.py".

BubuLK committed:
Misc: another tests try...

BubuLK committed:
Updated run_test.py

BubuLK committed:
Updated: run_test.py (#2)

BubuLK committed:
Updated: run_test.py (#3)

BubuLK committed:
Updated "test" section of meta.yaml.

BubuLK committed:
Removed mpi4py from run-requirements.

BubuLK committed:
Fixed meta.yaml syntax.

BubuLK committed:
Add examples dir to meta.yaml test section.

BubuLK committed:
Bump the build number.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

rc commented on README.md:
```diff
@@ -16,9 +16,9 @@ SfePy is a software for solving systems of coupled partial differential equation
-[![Windows](https://img.shields.io/appveyor/ci/conda-forge/sfepy-feedstock/master.svg?label=Windows)](https://ci.appveyor.com/project/conda-forge/sfepy-feedstock/branch/master)
+[![Linux](https://img.shields.io/circleci/project/github/conda-forge/SfePy-feedstock/master.svg?label=Linux)](https://circleci.com/gh/conda-forge/SfePy-feedstock)
+[![OSX](https://img.shields.io/travis/conda-forge/SfePy-feedstock/master.svg?label=macOS)](https://travis-ci.org/conda-forge/SfePy-feedstock)
+[![Windows](https://img.shields.io/appveyor/ci/conda-forge/SfePy-feedstock/master.svg?label=Windows)](https://ci.appveyor.com/project/conda-forge/SfePy-feedstock/branch/master)
```
Revert this, please, the paths are in lowercase.

BubuLK committed:
MNT: Re-rendered with conda-smithy 3.2.2 and pinning 2018.12.28

rc commented:
Maybe try using ``run_tests.py --filter-none`` option to see details printed by the failing tests? Or even ``--raise``?

BubuLK committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.12, and conda-forge-pinning 2019.01.29

BubuLK committed:
Removed duplicated file reference.

BubuLK committed:
Fix meta.yaml:
	- python build version (<3.7)
	- re-add run_tests.py
	- updated pytables build version

BubuLK committed:
Fix: meta.yaml #2.

BubuLK committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.12, and conda-forge-pinning 2019.01.29

BubuLK committed:
Add local conda_build_config.yaml

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * The top level meta key conda_build is unexpected

BubuLK committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.12, and conda-forge-pinning 2019.01.29

BubuLK committed:
Fix: meta.yaml #3.

BubuLK committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.12, and conda-forge-pinning 2019.01.29

BubuLK committed:
Fix: meta.yaml #4.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

BubuLK committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.12, and conda-forge-pinning 2019.01.29

BubuLK committed:
Fix: meta.yaml #5.

BubuLK committed:
Fix: meta.yaml #6.

rc commented:
The tests were added to 2019.1 branch and merged.
